pyproject.toml

```diff
@@ -24,25 +24,25 @@ django-modelcluster = "~6.0"
 django-pattern-library = "~1.0.0"
 django-permissions-policy = "~4.13.0"
 django-referrer-policy = "~1.0"
-django-storages = "~1.12.3"
+django-storages = "~1.13.1"
 django-taggit = "~2.1.0"
 djangorestframework = "~3.11.2"
 gunicorn = "~20.1.0"
 markdown = "~3.3.7"
-psycopg2 = "~2.9.3"
+psycopg2 = "~2.9.5"
 pygments = "~2.13.0"
 requests = "~2.28.1"
 scout-apm = "~2.26.1"
-sentry-sdk = "1.9.0"
-wagtail = "~4.0.2"
+sentry-sdk = "~1.10.1"
+wagtail = "~4.1.0"
 wagtail-font-awesome-svg = "~0.0.2"
 wagtailaltgenerator = "~4.1.2"  # Compatible with Wagtail <= 5.0 as far as I can tell, but not explicitly supported/tested by the package.
-wagtailmedia = "~0.11.0"
+wagtailmedia = "~0.12.0"
 whitenoise = "~6.2.0"
 django-redis = "~5.2"
 
 [tool.poetry.group.dev.dependencies]
-black = "22.8.0"
+black = "22.10.0"
 djhtml = "1.5.1"
 fabric3 = "1.14.post1"  # TODO switch to fabric/invoke
```

Comment:
Can we upgrade this as well? I just noticed `fab` commands are not working because there's an `from collections import Mapping` in Fabric that should be changed to `from collections.abc import Mapping` for Python 3.10

Reply:
hmm, the fabric upgrade will need more sweeping changes.

In the meantime, we could use https://github.com/mathiasertl/fabric/pull/57

Reply:
Ah, that's unfortunate. Would be nice if we didn't have to use a fork, but if the changes needed to properly upgrade are more involved, I guess it wouldn't hurt to switch to a fork. Not sure how far behind/ahead that branch in the `1.14.post1` version we use, though.

Reply:
it is quite up to date -- https://github.com/mathiasertl/fabric/compare/5913488a95b0318de0a862e7e181217d8a671641..HEAD